wondrous-app/components/Settings/Payouts/styles.tsx

```diff
@@ -24,23 +281,243 @@ export const TableCellText = styled(Typography)`
+  border-radius: 6px;
+  padding: 8px;
+  text-decoration: none;
+  transition: background 0.2s ease-in-out;
+
+  svg {
+    width: 16px;
+    height: 16px;
+  }
+
+  &:hover {
+    background: ${palette.highlightPurple}CC;
+  }
+`;
+
+export const PayoutTaskTitleContainer = styled.div`
+  padding: 6px 8px;
+  font-family: ${typography.fontFamily};
+  font-size: 13px;
+  background: ${palette.black92};
+  color: ${palette.white};
+  max-width: 175px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  border-radius: 6px;
+  position: relative;
+
+  &:after {
+    content: '';
+    display: block;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 40px;
+    height: 100%;
+    background: linear-gradient(to left, ${palette.black97}D9, ${palette.black92}3D);
+    z-index: 100;
+  }
+`;
+
+export const PayoutTaskCompletionDate = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 6px;
+  background: ${palette.black101};
+  border-radius: 6px;
+  padding: 6px 10px;
+
+  svg {
+    width: 16px;
+    height: 16px;
+  }
+`;
+
+export const PayoutTaskCompletionDateText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-size: 13px;
+    color: ${palette.white};
+    width: max-content;
+  }
+`;
+
+export const BottomActionBar = styled.div`
+  opacity: ${(props) => (props.isVisible ? 1 : 0)};
+  position: fixed;
+  bottom: ${(props) => (props.isVisible ? '60px' : '-100%')};
+  z-index: 100000;
+  left: 60%;
+  transform: translateX(-55%);
+  min-width: 800px;
+  width: 100%;
+  max-width: 960px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px 14px;
+  background: ${palette.black101};
+  border-radius: 6px;
+  transition: all 0.3s ease-in-out;
+`;
+
+export const BottomActionBarText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-size: 13px;
+    color: ${palette.grey250};
+  }
+`;
+
+export const BottomActionBarPayButton = styled(PayeePayButton)`
+  && {
+    button {
+      font-size: 15px;
+      padding: 8px 24px;
+      border-radius: 1000px;
+      background: ${palette.background.default};
+      transition: background 0.2s ease-in-out;
+
+      &:hover {
+        background: transparent;
+      }
+    }
+  }
+`;
+
+export const BottomActionBarButton = styled(Button)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-size: 15px;
+    font-weight: 500;
+    background: ${palette.grey78};
+    color: ${palette.white};
+    padding: 8px 24px;
+    border-radius: 1000px;
+
+    &:hover {
+      background: ${palette.grey85};
+    }
+  }
+`;
+
+export const BottomActionBarMultipleChainSelectedError = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  padding: 6px;
+  background: ${palette.grey85};
+  border-radius: 6px;
+
+  svg {
+    width: 20px;
+    height: 20px;
+  }
+`;
+
+export const BottomActionBarMultipleChainSelectedErrorText = styled(Typography)`
```

Comment:
use `sx` prop

Reply:
for this use case, it feels simplified to use a custom styled component